src/blockfilter.h

```diff
@@ -79,7 +85,7 @@ constexpr uint32_t BASIC_FILTER_M = 784931;
 
 enum BlockFilterType : uint8_t
 {
-    BASIC_FILTER = 0,
+    BASIC = 0,
```

Comment:
14172: Was this from another PR?

Reply:
Was part of the original implementation, see https://github.com/bitcoin/bitcoin/commit/c1855f6052aca806fdb51be01b30dfeee8b55f40#diff-130595b7adef87544bb3fb0cf726f4553768488234f8a26ec49fbb58fda4e517R81

Reply:
See 223dfc94dc18f52e6407bb941e962886a911a237

Reply:
Makes sense, cherry-picked!

Reply:
Hmm, that was not a commit to cherry-pick... 🙈  sorry, I wasn't clear enough about that. I mean, @PastaPastaPasta had issues back then which were fixed via 223dfc9. Is having `BASIC`  `BlockFilterType ` no longer an issue? If it's not - I'd prefer reverting the fix first and then applying backports on top of the revert commit cleanly instead of silently reverting the fix _inside_ of backports.

Reply:
What I meant by ^^^ is that this PR should look more like https://github.com/UdjinM6/dash/commits/pr4314 if that old issue is no longer reproducible. Can you confirm that `pr4314` branch works for you @PastaPastaPasta ?

Reply:
Resolved

Reply:
Seems fine, this branch compiles for me :)